modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/util/DDMIndexerImpl.java

```diff
@@ -647,6 +622,56 @@ private void _addFieldValueRequiredTerm(
 		}
 	}
 
+	private void _addFieldValueToStringBundler(
```

Comment:
@rolandpakai what about the other field types that aren't listed here?? Like `Numeric`, `Color`, `Date`, `Radio`...

Reply:
In case of **Numeric, Date, Boolean** field types this else statements will apply:
https://github.com/liferay/liferay-portal/blob/master/modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/util/DDMIndexerImpl.java#L374-L391

In case of **Color** field type the valueString is the color hex code.

We dont have Radio field. It is the Boolean field.

Reply:
@rolandpakai the Radio field is the Single Selection field. 

And how will it work for the color field (and others) if we don't have a `if (type.equals(DDMFormFieldTypeConstants.COLOR))`  in `_addFieldValue`, for example.

Reply:
I added the following else statement in the `_addFieldValue`
```
else {
			sb.append(valueString);
		}
```